spec/src/main/asciidoc/jakarta-ee.adoc

```diff
@@ -1,28 +1,78 @@
+CDI Full and CDI Lite approaches enable seamless integration between CDI and Jakarta EE Data. They provide mechanisms to produce and manage repository instances, ensuring that the desired implementation is available for injection when using the `@Inject` annotation.
 
-Jakarta Data providers that define custom entity annotations must follow the convention that the class name of all supported entity annotation types ends with `Entity`. This enables Jakarta Data providers to identify if a repository entity class contains entity annotations from different Jakarta Data providers so that the corresponding `Repository` can be ignored by Jakarta Data providers that should not provide it.
+By leveraging CDI extensions and the provided strategies, you can effectively integrate Jakarta EE Data with CDI and take advantage of its powerful dependency injection capabilities, promoting modular and reusable code in y
 
-Jakarta Data provider CDI Extensions must ignore all `Repository` annotations where annotations for the corresponding entity are available at run time and none of the entity annotations are supported by the Jakarta Data provider. Ignoring these `Repository` annotations allows other Jakarta Data providers to handle them.
+=== Mapping an Entity
+
+A Jakarta Data provider is an implementation of Jakarta Data that can utilize existing or new annotations for mapping entities. An entity refers to a class that is annotated to indicate its ability to represent objects in a storage engine, such as SQL or NoSQL databases.
+
+For instance, a Jakarta Data provider with Jakarta Persistence support will scan Java classes that are annotated with `jakarta.persistence.Entity`. On the other hand, a provider with Jakarta NoSQL integration will scan classes that are annotated with `jakarta.nosql.Entity`.
+
+It is important to note that a provider should ignore any annotations it does not support. For example, Jakarta Data NoSQL will ignore the Jakarta Persistence annotations and vice versa. By disregarding these `Repository` annotations, other Jakarta Data providers can handle them accordingly.
 
 ==== Jakarta Data Provider Name
 
-The entity annotation class will usually be sufficient to avoid conflicts between Jakarta Data providers, but in cases where the entity annotation class is not sufficient, the application can designate the name of the desired Jakarta Data provider on the optional `provider` attribute of the `Repository` annotation.
+The `entity` annotation class is the primary strategy to avoid conflicts between Jakarta Data providers. In most cases, it is sufficient to differentiate between providers. However, there may be situations where more than the entity annotation class is needed. In such cases, the application can specify the name of the desired Jakarta Data provider using the optional `provider` attribute of the `Repository` annotation.
```

Comment:
```suggestion
The entity annotation class is the primary strategy to avoid conflicts between Jakarta Data providers. In most cases, it is sufficient to differentiate between providers. However, there may be situations where more than the entity annotation class is needed. In such cases, the application can specify the name of the desired Jakarta Data provider using the optional `provider` attribute of the `Repository` annotation.
```